### PR TITLE
Adding info on restricting API

### PR DESCRIPTION
**Pull request by crgrieve**
Hello :)

Adding info on the Content Delivery API protect option.

Thanks,
Carole

### CODE REVIEW

File: Umbraco-Heartcore/Getting-Started-Cloud/Backoffice-Users-and-API-Keys/index.md [hunk 1]
```diff
@@ -15,7 +15,12 @@ A feature that is unique for Umbraco Heartcore is the option to create an API Ke
 
 ## API Keys
 
-In order to connect to the two APIs your user will need to have an API Key assinged. When you have navigated to the section mentioned above you can create the API Key by clicking the "Create API Key" button.
+By default, the Content Delivery API isn't restricted, if you would like to protect it you can do so in the "Settings" section and "Headless" tree. 
+
+![Protect API](images/heartcoreAPIprotect.png)
+
+
+If your Content Delivery API is protected or you want to use Content Management APIs, your user will need to have an API Key assinged. When you have navigated to the Users section mentioned above you can create the API Key by clicking the "Create API Key" button.
```
Comment (sofietoft) [inline Umbraco-Heartcore/Getting-Started-Cloud/Backoffice-Users-and-API-Keys/index.md]:
```suggestion
If your Content Delivery API is protected or you want to use Content Management APIs, your user will need to have an API Key assigned. When you have navigated to the Users section mentioned above you can create the API Key by clicking the "Create API Key" button.
```

**Comment by sofietoft** [comment]
Oh, I did add one tiny suggestion. Just a little spelling error! 😁

**Comment by sitereactor** [comment]
I'd like to change this
"By default, the Content Delivery API isn't restricted"
to
"By default, the Content Delivery API is public"

**Comment by crgrieve** [comment]
@sitereactor , I have updated that now :)

**Comment by sofietoft** [comment]
Thanks again @crgrieve 😄 

I've merged in the updates!

**Comment by crgrieve** [comment]
Awesome, thanks @sofietoft  (and @sitereactor).

Have a nice weekend! 😊

**Pull request merged**